core/frontend/src/components/parameter-editor/ServoFunctionLightsEditor.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div class="d-flex align-center">
+    <v-row>
+      <v-col cols="6">
+        <span class="text-h6">Joystick Steps</span>
+      </v-col>
+      <v-col cols="6">
+        <inline-parameter-editor
+          :label="steps_param?.name"
+          :param="steps_param"
+        />
+      </v-col>
+      <servo-function-range-editor
```

Comment:
This seems to be only partially implemented (there are no linked servo channel variables?), and weird because it's on one side:
<img width="466" height="278" alt="Image" src="https://github.com/user-attachments/assets/a8803c4b-cfbd-4cbe-87d5-5a2bb3f7577a" />